### DISCUSSION TITLE
政治冷感

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1868] [General]
各位知道1/13是什麼日子嗎？就是我們的總統大選！身為一位20歲的年輕人，要投出自己人生中的第一票，解鎖新體驗。

但其實我是一個政治冷感的人，即使政治等於我們年輕一代的未來，我還是沒什麼感覺，雖然這樣不太好，畢竟有自己的想法跟會批判思考各個候選人的政見是非常重要的，所以以後可能會試著多關心政治話題。

雖然我現在生活中最接近政治的是夜夜秀，但其實還是詼諧居多，但真的很好笑XD 不知道大家這次會不會回家投票，又會投給誰呢？

**Comment by BeaGle0928** [comment]
我曾經聽過一個說法，對於一個二十歲的學生而言，政治或許看起來遙不可及，也因此大部分二十出頭歲的人都覺得投不投票對於自己的生活好像沒有太大的影響，但到了三十甚至四十歲，開始認知到這些候選人的政策會對自己的生活造成很大的影響，要成家立業、買車買房，以及種種的生活開銷等等，到了這個時候人們才開始認真的看待選舉這件事，所以我認為在二十歲開始放一些些的心力在關心政治這門學問，在選舉的時候展現公民投票權，這些小小的舉動對於台灣，我們努力想要維護的民主自由的國家其實都有著深遠的影響。

**Comment by YANRONG0123** [comment]
我對於政治也完全沒什麼樣的感受，由於家鄉離學校過遠，所以我也不想回家投票什麼的，只要不會嚴重影響到我生活的選舉，我應該都是不支持也不反對，花費大量的時間或金錢，只是為了實現自己的某一項權利，我覺得不太值得。

**Comment by 20050201Shawn** [comment]
我自己也是比較偏政治冷感，有時甚至會覺得，就算理想中的候選人上台了，後續他真的會實踐他發表的政見嗎?可能這是所有民主國家的通病吧，現階段還是偏沒什麼特別的想法(其實我根本沒有投票權)

**Comment by SusuTima** [comment]
我本身也是比較政治冷感的人，與其相信在位者做出什麼大建設改變我從今以後的生活，我更願意多花一點時間在自己身上，不過po文的人說的的確有道理，政治人物的政策在未來確實會影響我們的生活，有時間還是要學著慢慢去關心政治。

**Comment by KH1126** [comment]
我也是政治冷感。
即使最近兩次大選我都有去投票，但我覺得自己的一票根本改變不了什麼，還不如不要在上面浪費自己的心智，多關注一些自己可以改變的事情。
